lively-lions/MUD/dungeon/views/character_view.py

```diff
@@ -0,0 +1,184 @@
+# from django.shortcuts import render
+from django.http import HttpResponse
+# from django.contrib.auth.decorators import login_required
+from dungeon.models.character import MudUser, Character
+from django.views import View
+from django.db import IntegrityError
+from django.db.models import F
+
+
+class CharacterView(View):
```

Comment:
nice to see class-based refactored views. It's the best solution to a very "fat" view in Django.